x-pack/plugins/synthetics/server/alert_rules/tls_rule/tls_rule_executor.ts

```diff
@@ -55,9 +55,10 @@ export class TLSRuleExecutor {
   }
 
   async getMonitors() {
+    const HTTP_OR_TCP = `${monitorAttributes}.${ConfigKey.MONITOR_TYPE}: http or ${monitorAttributes}.${ConfigKey.MONITOR_TYPE}: tcp`;
     this.monitors = await getAllMonitors({
       soClient: this.soClient,
-      filter: `${monitorAttributes}.${AlertConfigKey.TLS_ENABLED}: true`,
+      filter: `${monitorAttributes}.${AlertConfigKey.TLS_ENABLED}: true and (${HTTP_OR_TCP})`,
```

Comment:
Do we need to add the same monitor type filter in 8.8 an 8.9 migrations? Currently they add the flags to all monitor types.

Reply:
i already added those.